Makefile

```diff
@@ -1,242 +1,487 @@
 ##### PATHS #####
 
+VERSION=v1.4
+BASE_ENV_VERSION=v1.4
+PROJECT_ID=neuro-project-0f399af6
+
 DATA_DIR?=data
-CODE_DIR?=midi-generator
```

Comment:
This change does not relate to new makefile version. And it must be tested.

Reply:
It does: I had to rename code directory `midi-generator` -> `midi_generator`, otherwise it does not conform pattern of python package name and post-cookiecutter hooks fail.

Reply:
Ok, have you tried to run a job on the platform?)

Reply:
sure, I tested `make jupyter`. `make generate` fails with some error, I need help there. After that, needto test `make train`